docker-compose.dev.yml

```diff
@@ -28,8 +28,15 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock:ro
 
   app-db:
-    image: docker.io/library/postgres:13.2
+    profiles:
+      - dev
+      - api-only
+      - ui-only
+      - debug-api
+      - pg-only
+    image: docker.io/library/postgres:16
```

Comment:
Don't forget to update this in the `docker-compose.test.yml` as well if we decide to go with the version upgrade

Reply:
Just made the changes!